recipes/magnon/recipe.yaml

```diff
@@ -0,0 +1,60 @@
+context:
+  name: magnon
+  version: "1.0.0"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/8e/81/eb86a9e28708859cdbbc7dcf6f4bc40ce83343cd6f8b3a8177b5678f052c/magnon-1.0.0.tar.gz
+  sha256: 2f749c4fdd68c7c36162e0f0abcb6e1765f900ef680b746c57324878d7aa85a8
+
+build:
+  script: python -m pip install . -vv
+  number: 0
+  skip:
+     - match(python, "<3.12")
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - numpy
+    - matplotlib
+    - ase
+    - spglib
+    - pytest
```

Comment:
Is pytest really needed at build time? Actually, unless the package imports itself at build time, none of the run time dependencies are needed here.

```suggestion
```